<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineViewWidget</class>
 <widget class="QWidget" name="LineViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>25</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameL">
       <property name="text">
        <string>Контур:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="useChB">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="colorFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>23</horstretch>
         <verstretch>23</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorPB">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lineWidthL">
       <property name="text">
        <string>Ширина:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lineWidthCB"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="alphaL">
       <property name="text">
        <string>Прозрачность:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="alphaSB">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icegui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useChB</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LineViewWidget</receiver>
   <slot>useChB_stateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorPB</sender>
   <signal>clicked()</signal>
   <receiver>LineViewWidget</receiver>
   <slot>colorPB_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alphaSB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>LineViewWidget</receiver>
   <slot>alphaSB_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>colorPB_clicked()</slot>
  <slot>useChB_stateChanged(int)</slot>
  <slot>alphaSB_valueChanged(int)</slot>
 </slots>
</ui>
